### PR TITLE
Define GitHub action for build/test/publish 

### PR DESCRIPTION
**Pull request by dgrove-oss**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      # Build this repository
+      - name: Build Runtime
+        working-directory: runtime
+        run: |
+          ./gradlew distDocker
+
+      # Test this repository
+      - name: Test Runtime
+        working-directory: runtime
+        # TODO: we should also be doing  `./gradlew :tests:checkScalafmtAll`
```
Comment (dgrove-oss) [inline .github/workflows/ci.yaml:74]: Would prefer to also fix the formatting, but I don't have a Java11 environment installed on my current laptop, so grade doesn't work for me locally.

Comment (dgrove-oss) [reply .github/workflows/ci.yaml]: Later fixed it....

**Comment by dgrove-oss** [comment]
Thanks @msciabarra, but I think you need to approve using your GitHub that is associated with Apache for the approval to "count"

**Review by style95 — APPROVED** [review]

**Comment by msciabarra** [comment]
ops

**Pull request merged**